spec/ndx-extracellular-channels.extensions.yaml

```diff
@@ -0,0 +1,287 @@
+groups:
+- neurodata_type_def: ContactsTable
+  neurodata_type_inc: DynamicTable
+  default_name: contacts_table
+  doc: Metadata about the contacts of a probe, compatible with the ProbeInterface
+    specification.
+  datasets:
+  - name: relative_position_in_mm
+    neurodata_type_inc: VectorData
+    dtype: float
+    dims:
+    - - num_contacts
+      - x, y
+    - - num_contacts
+      - x, y, z
+    shape:
+    - - null
+      - 2
+    - - null
+      - 3
+    doc: Relative position of the contact in millimeters, relative to `reference`.
+    attributes:
+    - name: reference
+      dtype: text
+      doc: Reference point for the relative position coordinates and information about
+        the coordinate system used.
+      required: false
+  - name: shape
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: Shape of the contact; e.g. 'circle'
+  - name: contact_id
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: Unique ID of the contact
+    quantity: '?'
+  - name: shank_id
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: Shank ID of the contact
+    quantity: '?'
+  - name: plane_axes
+    neurodata_type_inc: VectorData
+    dtype: float
+    dims:
+    - - num_contacts
+      - v1, v2
+      - x, y
+    - - num_contacts
+      - v1, v2
+      - x, y, z
+    shape:
+    - - null
+      - 2
+      - 2
+    - - null
+      - 2
+      - 3
+    doc: The axes defining the contact plane. See 'contact_plane_axes' in https://probeinterface.readthedocs.io/en/main/format_spec.html
+      for more details.
+    quantity: '?'
+  - name: radius_in_um
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Radius of a circular contact, in micrometers.
+    quantity: '?'
+  - name: width_in_um
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Width of a rectangular or square contact, in micrometers.
+    quantity: '?'
+  - name: height_in_um
+    neurodata_type_inc: VectorData
+    dtype: float
+    doc: Height of a rectangular contact, in micrometers.
+    quantity: '?'
+  - name: device_channel_index_pi
```

Comment:
I read this as 'device channel index $\pi$'

Is that the intention?

If 'pi' is abbreviated for ProbeInterface for some reason, I would not abbreviate and just give the full context

Reply:
From what I can tell, this is the index of the channel connected to the contact on the device. I changed the name to just `device_channel`.

Reply:
Initially we used the `pi` suffix because everything that ends with `index` is handled differently